Author: Ole Streicher <olebole@debian.org>
Description: Disable test_no_numpy_warnings
 This test may fails when numpy decides that it wants to put some important
 message as a warning; even if nothing goes wrong. This happens f.e. for
 numpy-1.11.0~b2.
--- a/wcsaxes/tests/test_misc.py
+++ b/wcsaxes/tests/test_misc.py
@@ -51,6 +51,8 @@
 COORDSYS= 'icrs    '
 """, sep='\n')
 
+# see https://github.com/astrofrog/wcsaxes/issues/187
+@pytest.mark.xfail
 def test_no_numpy_warnings():
 
     # Make sure that no warnings are raised if some pixels are outside WCS
